.go-version

```diff
@@ -1 +1 @@
-GOVERS = go1.6.2
+GOVERS = (go1.6.2|go1.7rc1)
```

Comment:
Heh, this shows up in the error message, so it seemed better to make it slightly more intelligible. I was imagining we'd update this when further release candidates are released. Doesn't seem onerous to do so.